### ISSUE TITLE
listitem should not be able to be disabled

### ISSUE THREAD

**Issue by sh977218** [twbs/bootstrap#40752]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

this page: https://getbootstrap.com/docs/5.3/components/list-group/#disabled-items
it has a `ul` with `li` being `aria-disabled="true"

```html
<ul class="list-group">
  <li class="list-group-item disabled" aria-disabled="true">A disabled item</li>
  <li class="list-group-item">A second item</li>
  <li class="list-group-item">A third item</li>
  <li class="list-group-item">A fourth item</li>
  <li class="list-group-item">And a fifth one</li>
</ul>
```

however, in https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled#associated_roles
mozilla does not have `aria-disabled` to the 'li' which role is `listitem`..

### Reduced test cases

```html
<ul class="list-group">
  <li class="list-group-item disabled" aria-disabled="true">A disabled item</li>
  <li class="list-group-item">A second item</li>
  <li class="list-group-item">A third item</li>
  <li class="list-group-item">A fourth item</li>
  <li class="list-group-item">And a fifth one</li>
</ul>
```

### What operating system(s) are you seeing the problem on?

Windows, macOS

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5

**Comment by Amenawon** [comment]
It looks like aria-disabled are more suited for buttons than a list item, just a class should suffice.
Can I please be assigned to this ticket?
